destination/spool.go

```diff
@@ -88,7 +88,7 @@ func (s *Spool) Reader() {
 	ch := s.Out
 	queue := s.queue
 
-	h := encoding.NewPlain(false)
+	h := encoding.NewPlain(false, true)
```

Comment:
So in case of spooling we loose the tags ?

Reply:
Default value is indeed True: we drop all tags. It will be set to False (to be precised, a new structure will be used) if we need to keep tags.